python/neutron-understack/pyproject.toml

```diff
@@ -13,5 +38,16 @@ select = [
   "ASYNC",   # flake8-async
 ]
 
-[tool.ruff.lint.isort]
-force-single-line = true
+ignore = [
+    "D100",  # don't require docs for every module
+    "D101",  # don't require docs for every class
+    "D102",  # don't require docs for every class method
+    "D103",  # don't require docs for every function
+    "D104",  # don't require docs for every package
+    "D417"   # don't require docs for every function parameter
+]
+
+[tool.ruff.lint.pydocstyle]
+# enable the google doc style rules by default
+convention = "google"
+
```

Comment:
can we carry over the `classifiers` too?

```toml
classifiers = [
    "Development Status :: 3 - Alpha",
    "Environment :: OpenStack",
    "Intended Audience :: System Administrators",
    "Intended Audience :: Information Technology",
    "License :: OSI Approved :: Apache Software License",
    "Operating System :: OS Independent",
    "Programming Language :: Python",
    "Programming Language :: Python :: Implementation :: CPython",
    "Programming Language :: Python :: 3 :: Only",
    "Programming Language :: Python :: 3",
    "Programming Language :: Python :: 3.10",
]
```

Reply:
Done in 5e711c7